### DISCUSSION TITLE
flutter 什么时候考虑支持?

### DISCUSSION THREAD

**Discussion by git91895** [Tencent/libpag#280] [Ideas] [answered]
flutter 什么时候考虑支持?

**Comment by kevingpqi123** [comment, marked as answer]
关于Flutter的相关回复，可以看FAQ：
【PAG 支持 Flutter 使用，目前已经有成功使用的案例，需要业务层做些适配。具体原理是通过外部纹理做桥接，PAG 可以将动画逐帧绘制到指定的纹理上。因此，外部的任何框架包括 Flutter 只要能提供一个纹理给 PAG 绘制，都可以完美实现桥接或融合。具体请搜索 Flutter 外接纹理接口，提供一个纹理包装成 PAGSurface后，然后使用 PAGPlayer 正常进行绘制控制即可。】
对于我们自身的业务而言，暂时没有Flutter的使用场景，也不太清楚大家需要或者适合大家的封装形式是什么，大家可以在pag的基础上做一些封装分享出来，我们可以帮忙推广
